pyplot/baselineFits/ModuleMapper.py

```diff
@@ -0,0 +1,134 @@
+    string_to_hw["L4B_stereo_ele"] = "F5H0"
+    string_to_hw["L4B_stereo_pos"] = "F5H1"
+    string_to_hw["L4B_axial_ele" ] = "F5H2"
+    string_to_hw["L4B_axial_pos" ] = "F5H3"
+    string_to_hw["L5T_axial_ele" ] = "F6H0"
+    string_to_hw["L5T_axial_pos" ] = "F6H1"
+    string_to_hw["L5T_stereo_ele"] = "F6H2"
+    string_to_hw["L5T_stereo_pos"] = "F6H3"
+    string_to_hw["L5B_stereo_ele"] = "F7H0"
+    string_to_hw["L5B_stereo_pos"] = "F7H1"
+    string_to_hw["L5B_axial_ele" ] = "F7H2"
+    string_to_hw["L5B_axial_pos" ] = "F7H3"
+    string_to_hw["L6T_axial_ele" ] = "F8H0"
+    string_to_hw["L6T_axial_pos" ] = "F8H1"
+    string_to_hw["L6T_stereo_ele"] = "F8H2"
+    string_to_hw["L6T_stereo_pos"] = "F8H3"
+    string_to_hw["L6B_stereo_ele"] = "F9H0"
+    string_to_hw["L6B_stereo_pos"] = "F9H1"
+    string_to_hw["L6B_axial_ele" ] = "F9H2"
+    string_to_hw["L6B_axial_pos" ] = "F9H3"
+    return string_to_hw[string]
+
+def str_to_sw(string):
+    string_to_sw = {}
+    string_to_sw["L0T_axial"     ] = "ly1_m0" 
+    string_to_sw["L0T_stereo"    ] = "ly2_m0" 
+    string_to_sw["L1T_axial"     ] = "ly3_m0" 
+    string_to_sw["L1T_stereo"    ] = "ly4_m0" 
+    string_to_sw["L1B_axial"     ] = "ly4_m1" 
+    string_to_sw["L1B_stereo"    ] = "ly3_m1" 
+    string_to_sw["L0B_axial"     ] = "ly2_m1" 
+    string_to_sw["L0B_stereo"    ] = "ly1_m1" 
+    string_to_sw["L2T_axial"     ] = "ly5_m0" 
+    string_to_sw["L2T_stereo"    ] = "ly6_m0" 
+    string_to_sw["L3T_stereo"    ] = "ly8_m0" 
+    string_to_sw["L3T_axial"     ] = "ly7_m0" 
+    string_to_sw["L2B_stereo"    ] = "ly5_m1" 
+    string_to_sw["L2B_axial"     ] = "ly6_m1" 
+    string_to_sw["L3B_stereo"    ] = "ly7_m1" 
+    string_to_sw["L3B_axial"     ] = "ly8_m1" 
+    string_to_sw["L4T_axial_ele" ] = "ly9_m0" 
+    string_to_sw["L4T_axial_pos" ] = "ly9_m2" 
+    string_to_sw["L4T_stereo_ele"] = "ly10_m0"
+    string_to_sw["L4T_stereo_pos"] = "ly10_m2"
+    string_to_sw["L4B_stereo_ele"] = "ly9_m1" 
+    string_to_sw["L4B_stereo_pos"] = "ly9_m3" 
+    string_to_sw["L4B_axial_ele" ] = "ly10_m1"
+    string_to_sw["L4B_axial_pos" ] = "ly10_m3"
+    string_to_sw["L5T_axial_ele" ] = "ly11_m0"
+    string_to_sw["L5T_axial_pos" ] = "ly11_m2"
+    string_to_sw["L5T_stereo_ele"] = "ly12_m0"
+    string_to_sw["L5T_stereo_pos"] = "ly12_m2"
+    string_to_sw["L5B_stereo_ele"] = "ly11_m1"
+    string_to_sw["L5B_stereo_pos"] = "ly11_m3"
+    string_to_sw["L5B_axial_ele" ] = "ly12_m1"
+    string_to_sw["L5B_axial_pos" ] = "ly12_m3"
+    string_to_sw["L6T_axial_ele" ] = "ly13_m0"
+    string_to_sw["L6T_axial_pos" ] = "ly13_m2"
+    string_to_sw["L6T_stereo_ele"] = "ly14_m0"
+    string_to_sw["L6T_stereo_pos"] = "ly14_m2"
+    string_to_sw["L6B_stereo_ele"] = "ly13_m1"
+    string_to_sw["L6B_stereo_pos"] = "ly13_m3"
+    string_to_sw["L6B_axial_ele" ] = "ly14_m1"
+    string_to_sw["L6B_axial_pos" ] = "ly14_m3"
+    return string_to_sw[string]
+
+def hw_to_str(string):
```

Comment:
Instead of making this list, it would be much better to invert the str_to_hw() string using a loop.